h2o-hadoop-3/assemblyjar.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
     api project(":h2o-ext-steam")
     api "org.apache.hadoop:hadoop-client:$hadoopMavenArtifactVersion"
     api "org.apache.hadoop:hadoop-mapreduce-client-app:$hadoopMavenArtifactVersion"
+    api "org.apache.hadoop:hadoop-aws:$hadoopMavenArtifactVersion"
```

Comment:
This one is defined in `assemblyjar_common.gradle` (because it is shared for CDP and other hadoop distros), we do not bundle any dependencies for CDP - that is why it is different.

Reply:
I see, thank you!